docs/content/en/cookbook-core.md

```diff
@@ -0,0 +1,363 @@
+Our first module is a required module, so we also add the module name to the `required` property.
+
+```javascript [nuxt.config.js]
+  export default {
+    vuems: {
+      modules: {
+        local: ['@example/core'],
+      },
+      required: ['@example/core'],
+      logLoadedModules: true,
+    },
+  }
+```
+
+## Component
+
+Create the first components in `components` directory.
+
+### `Logo.vue`
+Create a `Logo.vue` file in `component` directory.
+
+
+<alert type="warning">
+
+  I am not presenting a full example of the `Logo.vue` component because it contains the `svg` file content.
+
+  If you want to see the full content of the component, you can find it in the
+  <a href="https://github.com/ergonode/vuems-demo" target="_blank">repository</a>.
+
+</alert>
+
+```vue [@Core/components/Logo.vue]
+<template>
+  <svg> ... </svg>
+</template>
+
+<script>
+export default {
+  name: 'Logo',
+}
+</script>
+
+<style>
+.VuemsLogo {
+  animation: 1s appear;
+  margin: auto;
+  width: 300px;
+}
+@keyframes appear {
+  0% {
+    opacity: 0;
+  }
+}
+</style>
+```
+
+### `Wrapper.vue`
+Create a `Wrapper.vue` file in `component` directory.
+The component will wrap other components.
+
+```vue [@Core/components/Wrapper.vue]
+<template>
+  <div class="wrapper">
+    <Logo />
+    <slot />
+  </div>
+</template>
+
+<script>
+import Logo from '@Core/components/Logo'
+
+export default {
+  name: 'Wrapper',
+  components: {
+    Logo,
+  },
+}
+</script>
+
+<style>
+.wrapper {
+  margin: 0 auto;
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+</style>
+```
+
+### `Navigation.vue`
+Create a `Navigation.vue` file in `component` directory.
+The component will be used in the layout component.
+
+```vue [@Core/components/Navigation.vue]
+<template>
+  <nav class="navigation">
+    <ul>
+      <li><NuxtLink class="button--green" to="/">Home page</NuxtLink></li>
+      <li><NuxtLink class="button--grey" to="/about">About</NuxtLink></li>
+    </ul>
+  </nav>
+</template>
+<script>
+export default {
+  name: 'Navigation',
+}
+</script>
+
+<style>
+.button--green {
+  display: inline-block;
+  border-radius: 4px;
+  border: 1px solid #3b8070;
+  color: #3b8070;
+  text-decoration: none;
+  padding: 10px 30px;
+}
+
+.button--green:hover {
+  color: #fff;
+  background-color: #3b8070;
+}
+
+.button--grey {
+  display: inline-block;
+  border-radius: 4px;
+  border: 1px solid #35495e;
+  color: #35495e;
+  text-decoration: none;
+  padding: 10px 30px;
+  margin-left: 15px;
+}
+
+.button--grey:hover {
+  color: #fff;
+  background-color: #35495e;
+}
+
+.navigation {
+  position: absolute;
+  padding-top: 15px;
+}
+
+.navigation ul {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+}
+</style>
+
+```
+
+## Page
+
+The next thing is to add a new page.
+We create a `pages` directory where we add the page.
+
+In the new page component, we add our previously created `Wrapper` component.
```

Comment:
There is no need to repeat info about `Wrapper` component. Consider to add Chapter with: What you need at the start to make thing happening.